cypress/e2e/enable-workspaces-alert.cy.ts

```diff
@@ -0,0 +1,27 @@
+describe('Enable Workspaces Alert', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/iam/user-access/overview');
+    cy.wait(5000);
+  });
+
+  it('should be visible', () => {
+    cy.get('[data-ouia-component-id="enable-workspaces-alert"]').should(
+      'include.text',
+      'You are qualified to opt into the workspace user access model for your organization.'
+    );
+  });
+
+  it('should open Enable Workspaces Modal', () => {
+    cy.get('[data-ouia-component-id="enable-workspaces-switch"]').click();
+    cy.get('[data-ouia-component-id="enable-workspaces-modal-header"]').should('have.text', 'Enable workspaces');
+  });
+
+  it('should show success alert after confirming', () => {
+    cy.wait(3000);
```

Comment:
the page loads a bit slow sometimes

Reply:
We should replace cy.wait with cy.intercept - this waits for a network response instead of a fixed time, making the tests faster and more reliable.